gp-includes/things/translation.php

```diff
@@ -252,6 +252,15 @@ public function prepare_fields_for_save( $args ) {
 		if ( is_array( gp_array_get( $args, 'warnings' ) ) ) {
 			$args['warnings'] = serialize( $args['warnings'] );
 		}
+		/**
+		 * Fires the translation is preparing for save.
```

Comment:
I think this may have reverted back at some point, should be "Filters the" instead of "Fires the".